### PR TITLE
 TA - fix rubocop styling for static home page controller

### LINKED ISSUE #1: Deployment Environment
#### As a developer I want a continuous deployment environment setup for my new Rails application.

__Story__:

Hi! I’ve heard you want to build an image sharing website using
ruby-on-rails. That’s wonderful! You have quite the adventure ahead of you
while you build out this working site.

To help you get started, I have already created a source repository for
you. This repository, on GitHub, contains the source code, and the list of
issues, including this one, that you will work on as you build out the features
for your website.

While you build out features, you want to ensure you do not break
anything. Thus with each feature you write, you will also write the necessary
model, controller, and flow tests. While you should always run your tests
locally before committing code, we have configured CircleCI as our automated
continuous integration system; that is, CircleCI will run all the tests on a
branch whenever you push a modification to it on GitHub.

Finally, you have a complete deployment pipeline set up on Heroku. With this
pipeline, there are three steps used for testing your application. Review
applications are automatically created whenever you make a pull request. There
can be as many review applications as you have open pull requests. This step
enables you to perform quality assurance (QA) tests while you are developing a
feature. The second step, staging, is used to ensure no bugs are introduced as
the result of a feature branch merge into master. The staging server is as
close of a replica to your production server there is, so it provides a great
way to ensure that changes will not negatively impact your customers. The
staging server is automatically deployed whenever the master branch is updated
and the automated tests pass. The last step is production. This step contains
your production application, and the goal is for that server to have zero
unscheduled downtime – all of the other steps work toward that effort.

Please keep in mind that each feature delivers a modicum of value to either
your users, or to aid in your development of the website. You want to deliver
value to your users fast, so some features, like authentication, are saved
until your users really need them.

To begin with this story, please verify that things are set up for you by
checking off each item below. If anything does not appear to work, it’s up to
you to fix it and make it work.

I encourage you to read the discussion topics and keep them in mind when
working on the story. You should be able to provide answers to them when the
story is complete.

The references might provide insight into the discussion topics, help you
complete the story, or simply provide more information that you might be
interested in. It is not expected that you read through all the references --
you can if you want -- however, it might be a good idea to scan through them
before getting started with a story.

Happy coding!

__Acceptance criteria__:
- [x] [GitHub](https://github.com/AppFolioOnboarding) is used for source
  control.
- [x] The project's version of both Ruby and Rails are used both locally and on
  Heroku.
- [x] The landing page, `/`, provides some content.
- [x] Every available controller action has corresponding tests.
- [x] [CircleCI](https://circleci.com/) is used for Continuous Integration.
- [x] Heroku is used to deploy the application.
- [x] Pull requests are automatically deployed as review applications.
- [x] The staging application automatically deploys whenever the master branch
  is updated.
- [x] The staging application automatically deploys only after all tests pass
  on CircleCI.
- [x] The staging application deployment can be promoted to the production
  application deployment.
- [x] [Rubocop](https://github.com/bbatsov/rubocop) is used to help with code
  style consistency.

__Discussion Topics__:
- [x] Where can we discover how long it takes to return an HTTP response?
- [x] When should we commit our code?

__References__:
* https://rvm.io/rvm/install#try-out-your-new-rvm-installation
* http://guides.rubyonrails.org/getting_started.html
* http://guides.rubyonrails.org/testing.html#functional-tests-for-your-controllers
* https://devcenter.heroku.com/articles/getting-started-with-rails5
* https://devcenter.heroku.com/articles/pipelines
* https://github.com/bbatsov/ruby-style-guide

### PR DESCRIPTION
**Pull request by treyamador**
Closes #1 Rubocop stylings fixed static page controller to serve home page

### CODE REVIEW

File: app/controllers/pages_controller.rb [hunk 1]
```diff
@@ -1,5 +1,4 @@
 # a controller to serve (generally) static pages
 class PagesController < ApplicationController
-  def home
-  end
+  def home() end
```
Comment (JingyuZ) [inline app/controllers/pages_controller.rb]: It's minor, but in ruby you don't need to use parentheses, so this could be a one liner like `def home; end`.

File: Gemfile.lock [hunk 2]
```diff
@@ -245,4 +245,4 @@ RUBY VERSION
    ruby 2.5.1p57
 
 BUNDLED WITH
-   1.16.2
+   1.16.6
```
Comment (JingyuZ) [inline Gemfile.lock]: I'm not quite sure about what bundler version is used on your local machine vs heroku, the bundler version change doesn't seem like necessary to me and could lead other issues.

File: app/assets/javascripts/pages.js [hunk 3]
```diff
@@ -0,0 +1,2 @@
+// Place all the behaviors and hooks related to the matching controller here.
+// All this logic will automatically be available in application.js.
```
Comment (JingyuZ) [inline app/assets/javascripts/pages.js]: I think file was created by rails generator and we are actually not using it right now, so I would not include this file in this PR.

File: app/assets/stylesheets/pages.scss [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+body > .container {
+  padding: 60px 15px 0;
+}
+.container .text-muted {
+  margin: 20px 0;
+}
+
+.footer > .container {
+  padding-right: 15px;
+  padding-left: 15px;
```
Comment (JingyuZ) [inline app/assets/stylesheets/pages.scss]: Supor minor, you can also use `padding: 0 15px`, the first specifies vertical padding, and the second specifies horizontal padding.

File: app/helpers/pages_helper.rb [hunk 5]
```diff
@@ -0,0 +1,2 @@
+module PagesHelper
+end
```
Comment (JingyuZ) [inline app/helpers/pages_helper.rb]: Same as `page.js`, we don't need to commit things we don't need for now.

File: config/environments/development.rb [hunk 6]
```diff
@@ -1,6 +1,9 @@
 Rails.application.configure do
   # Verifies that versions and hashed value of the package contents in the project's package.json
-  config.webpacker.check_yarn_integrity = true
+  # config.webpacker.check_yarn_integrity = true
+
+  # added to prevent yarn from failing
+  config.webpacker.check_yarn_integrity = false
```
Comment (JingyuZ) [inline config/environments/development.rb]: Just curious, why did you add new stuff instead of updating line 3? It's a bit confusing...

Comment (treyamador) [reply config/environments/development.rb]: Yes, I missed that.  I had difficulty getting yarn to cooperate and was testing different parameters.  I had intended to remove that.

File: app/controllers/pages_controller.rb (hunk 1 above)
Comment (treyamador) [reply app/controllers/pages_controller.rb]: I was unaware of the precedent of using a semicolon.  Thanks for the tip.

File: Gemfile.lock (hunk 2 above)
Comment (treyamador) [reply Gemfile.lock]: I have no idea how or why that changed.  That was not deliberate.  I'll look for that in the future.

File: app/assets/javascripts/pages.js (hunk 3 above)
Comment (treyamador) [reply app/assets/javascripts/pages.js]: I'll note that for the future.

File: app/assets/stylesheets/pages.scss (hunk 4 above)
Comment (treyamador) [reply app/assets/stylesheets/pages.scss]: Yes, I should've done it that way.

File: app/helpers/pages_helper.rb (hunk 5 above)
Comment (treyamador) [reply app/helpers/pages_helper.rb]: I'll keep that in mind for the future.
